.editorconfig

```diff
@@ -0,0 +1,24 @@
+# editorconfig.org
+# editorconfig is a unified configuration file that all editors can take
+# into account
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.coffee]
```

Comment:
I will remove theses lines from the template : coffee, stylus, etc

Reply:
Done : https://github.com/konnectors/boulanger/pull/2/commits/a05adb7b524f4645267e9a9c290f0e93e3df2a82